### ISSUE TITLE
Page feedback re: load generator docs

### ISSUE THREAD

**Issue by szeitlin** [open-telemetry/opentelemetry.io#4977]
Suggested improvements for page: https://opentelemetry.io/docs/demo/services/load-generator/

This isn't really documentation, it feels like a stub. I can't tell if the load generator service is already included in the installation, or exactly what I need to do to get it running on my system. Where does the locustfile live? Do I need to generate one? How can I tell if it's working as expected?

**Comment by theletterf** [comment]
@open-telemetry/demo-approvers Any thoughts on this one? Thanks!

**Comment by rogercoll** [comment]
I agree that the documentation should be improved, the `locustfile` is [provided](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/loadgenerator/locustfile.py) and launched with the other OpenTelemetry services.

Some load is automatically generated, and the configuration can be manually changed in the `/loadgen` endpoint: https://github.com/open-telemetry/opentelemetry-demo/tree/main/src/loadgenerator#accessing-the-load-generator

> How can I tell if it's working as expected?

As the other services, by checking the `loadgen` container state. Another option is to check if the `/loadgen` endpoint is available.

**Comment by szeitlin** [comment]
> and launched with the other OpenTelemetry services.

This doesn't seem to be the case, and the `/loadgen` endpoint isn't accessible. So I'm wondering how to troubleshoot that. I tried editing the helm values to make sure the load generator service is enabled, but that didn't seem to be sufficient. I also tried passing that as a `--set` flag to helm, but I kept getting an error when I did that.

**Comment by szeitlin** [comment]
oh, I think maybe I need to give it more resources? Looks like it tried to come up and got stuck in pending for 24h:

`loadgenerator                 0/1     Pending            0                 24h`

**Comment by szeitlin** [comment]
ok, that wasn't the entire problem. I can load the webapp at `localhost:8080` but I'm getting a 404 at `localhost:8080/loadgen` 

Is it possible there's a port collision or there's a step missing with forwarding from another port-?

**Comment by szeitlin** [comment]
Indeed, the port this service wanted to use was `8089` not `8080` so I had to do an additional port forwarding step. But now it works!

**Comment by svrnm** [comment]
@szeitlin @rogercoll would you be interested in improving the docs accordingly? Thanks
